### PR TITLE
Like Post

### PR DESCRIPTION
**Pull request by skybluecf**
Link Redmine: https://edu-redmine.sun-asterisk.vn/issues/15420

### CODE REVIEW

**Comment by skybluecf** [comment]
Ready

File: app/views/likes/_like_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  <%= post.likes.count %><i class="fa fa-heart"></i>
+</div>
+<div id="like-form">
+  <% pre_like = post.likes.find { |like| like.user_id == current_user.id} %>
+  <% if pre_like %>
+    <%= render "likes/unlike", post: post, pre_like: pre_like %>
+  <% else %>
+    <%= render "likes/like", post: post %>
+  <% end %>
+</div>
```
Comment (hungct-1121) [inline app/views/likes/_like_form.html.erb]: thêm dòng trống cuối file

Comment (skybluecf) [reply app/views/likes/_like_form.html.erb]: Ợ 🤕

**Comment by skybluecf** [comment]
Ready

**Pull request merged**